omero/developers/Server/Clustering.rst

```diff
@@ -46,3 +46,50 @@ newly created server will not be used until sessions are manually
 redirected to it.
 
 .. seealso:: :doc:`/developers/Server/ScalingOmero`
+
+Read-Only
+---------
+
+A read-only server offers users the ability to do little more than log
```

Comment:
`to do little more than` this sounds unduly negative, since it's actually really powerful functionality that let's you scale OMERO to as many servers as you want for data retrieval.

Reply:
Ah, it was meant to sound reassuring, I clearly failed there!